docs/source/asr/ssl/results.rst

```diff
@@ -0,0 +1,108 @@
+
+Note that these instructions are for fine-tuning. To resume an unfinished training experiment, 
+use the Experiment Manager to do so by setting the ``resume_if_exists`` flag to ``True``.
+
+Loading Local Checkpoints
+-------------------------
+
+NeMo automatically saves checkpoints of a model that is trained in a ``.nemo`` format. Alternatively, to manually save the model at any 
+point, issue :code:`model.save_to(<checkpoint_path>.nemo)`.
+
+If there is a local ``.nemo`` checkpoint that you'd like to load, use the :code:`restore_from()` method:
+
+.. code-block:: python
+
+  import nemo.collections.asr as nemo_asr
+  ssl_model = nemo_asr.models.<MODEL_BASE_CLASS>.restore_from(restore_path="<path/to/checkpoint/file.nemo>")
+
+Where the model base class is the ASR model class of the original checkpoint, or the general ``ASRModel`` class.
+
+Loading NGC Pretrained Checkpoints
+--------------------------
+
+The SSL collection has checkpoints of several models trained on various datasets. These checkpoints are 
+obtainable via NGC `NeMo Automatic Speech Recognition collection <https://catalog.ngc.nvidia.com/orgs/nvidia/collections/nemo_asr>`_.
+The model cards on NGC contain more information about each of the checkpoints available.
+
+The table at the end of this page lists the SSL models available from NGC. The models can be accessed via the :code:`from_pretrained()` method inside
+the ASR Model class. In general, you can load any of these models with code in the following format:
+
+.. code-block:: python
+
+  import nemo.collections.asr as nemo_asr
+  ssl_model = nemo_asr.models.ASRModel.from_pretrained(model_name="<MODEL_NAME>")
+
+Where the ``model_name`` is the value under "Model Name" entry in the tables below.
+
+For example, to load the conformer Large SSL checkpoint, run:
+
+.. code-block:: python
+
+  ssl_model = nemo_asr.models.ASRModel.from_pretrained(model_name="ssl_en_conformer_large")
+
+You can also call :code:`from_pretrained()` from the specific model class (such as :code:`SpeechEncDecSelfSupervisedModel`
+for Conformer) if you need to access a specific model functionality.
+
+If you would like to programatically list the models available for a particular base class, you can use the
+:code:`list_available_models()` method.
+
+.. code-block:: python
+
+  nemo_asr.models.<MODEL_BASE_CLASS>.list_available_models()
+
+
+Loading SSL checkpoint into Down-stream Model
+-------------------------------------------
+After loading an SSL checkpoint as shown above, it's ``state_dict`` needs to be copied to a 
+down-stream model for fine-tuning. 
+
+For example, to load a SSL checkpoint for ASR down-stream task using ``EncDecRNNTBPEModel``, run:
+
+.. code-block:: python
+
+  # define down-stream model
+  asr_model = nemo_asr.models.EncDecRNNTBPEModel(cfg=cfg.model, trainer=trainer)
+
+  # load ssl checkpoint
+  asr_model.load_state_dict(ssl_model.state_dict(), strict=False)
```

Comment:
Hmm how about actually providing some more utility methods to stitch together weights, similar to whats done in init_from_pretrained()?

We also need higher order methods to "split state dict" so that this "strict=False" isn't needed and can safely be loaded to actual encoders.

Reply:
Thanks for the suggestion. Will plan for this in next update.